.vitepress/theme/Layout.vue

```diff
@@ -50,13 +60,15 @@
 </template>
 
 <script setup>
-import { computed } from 'vue';
-import { useRoute } from 'vitepress';
-import Posts from './Posts.vue';
-import Post from './Post.vue';
+import { computed } from "vue";
```

Comment:
ugh this all appears to have come from prettier

Reply:
Haha np